lib/samples/set_basemap/README.md

```diff
@@ -0,0 +1,29 @@
+# Set basemap
+
+Change a map's basemap. A basemap is beneath all layers on an `ArcGISMap` and is used to provide visual reference for the operational layers.
```

Comment:
```suggestion
Change a map's basemap. A basemap is beneath all layers on a map and is used to provide visual reference for the operational layers.
```
The design is wrong here, we shouldn't have classes referenced in the description. So just tweak to map. (I'll clean up the design).